core/test/integration/api/api_schedules_spec.js

```diff
@@ -179,7 +179,7 @@ describe('Schedules API', function () {
             originalCannotScheduleAPostBeforeInMinutes = config.times.cannotScheduleAPostBeforeInMinutes;
 
             // we can insert published_at less then 5minutes
-            config.times.cannotScheduleAPostBeforeInMinutes = -5;
+            config.times.cannotScheduleAPostBeforeInMinutes = -15;
```

Comment:
Tests shouldn't modify config directly. There is a utility for setting and restoring config, see `model_posts_spec.js` for a good example.